ixp_tracker/importers.py

```diff
@@ -115,7 +115,8 @@ def do_process_ixp_data(all_ixp_data):
                         "created": ixp_data["created"],
                         "last_updated": ixp_data["updated"],
                         "last_active": processing_date,
-                        "manrs_participant": ixp_data["id"] in manrs_lookup.get_manrs_participants(processing_date),
+                        "manrs_participant": ixp_data["id"] in data_lookup.get_manrs_participants(processing_date),
```

Comment:
suggestion: make one call outside the loop to get lists of MANRS participants and anchor hosts for performance reasons (I guess Django may cache the result but perhaps we shouldn't rely on that).